app/main.py

```diff
@@ -1,3 +1,54 @@
 class Distance:
     # Write your code here
-    pass
+    def __init__(self, km: (int, float)) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: (int, float, "Distance")) -> "Distance":
```

Comment:
Change your type annotation to use `|` and class without quotes in the return type annotation:
```suggestion
    def __add__(self, other: int | float | Distance) -> Distance:
```
Use `from __future__ import annotations` import in case you face errors. Change your type annotation in all places.